_drafts/asking-for-help.md

```diff
@@ -0,0 +1,40 @@
+As some of us find ourselves with extra time on our hands in the COVID-19 era many at Fred Hutch are looking to sharpen their programming skills. Luckily, we have some amazing opportunities and resources to learn digitally through The Coop, TDS-IRC, and FredHutch.io. A natural consequence of more people learning to code is that we've seen an increase in messages related to looking for coding help via slack channels, community groups, and personal messages to folks like Kate and myself. We've noticed a stark difference in how seasoned and newer programmers will structure their messages seeking help. Someone newer to programming may send a message that goes something like this: `Hi, I can't install tidyverse. Can you please help?`. While there is nothing technically wrong with asking for help this way - it will take a few back and forth messages, possibly running some code again, to diagnose the issue. In this post we will go over some norms and best-practices for asking for coding help through slack, email, or online forum.
+
+First, I want to note that a fear of asking for help can be compounded by very real oppressive structures and lived experience. Women, gender-diverse folks, and people of color experience vitriol online at higher rates. Further, these groups often encounter increased scrutiny at work that can make regularly asking for help more intimidating and possibly detrimental to career progress. It's important to cultivate a work environment that actively addresses power structures to create a space that encourages open, vulnerable communication and where asking for help feels comfortable to all.
+
+1. **Do some research before posting/asking**
+
+- The first step to encountering a problem is trying to solve it or at least understand it as much as you can on your own. Toss your error code on google, peruse help forums, try a few solutions. 
+
+- *Best case scenario:* Someone has already asked your exact question and the solution provided works for you! It's good practice to double-check that your question has not already been answered in an internet forum before posting.
```

Comment:
...in an Internet forum like [StackOverflow](https://stackoverflow.com) or the [GATK Community Forum](https://gatk.broadinstitute.org/hc/en-us/community/topics) before posting.